iDiff/fileDiff_test_processor.py

```diff
@@ -0,0 +1,35 @@
+import json
+import re
+import sys
+
+
+def _process_test_diff(file_path):
+    f = open(file_path)
+    diff = json.load(f)
```

Comment:
if the call to `json.load()` fails for some reason, this file descriptor won't be closed. if you wrap this in a `with open(file_path) as f:` block, that will automatically close the file before the program exits.